soyiyeon/백준_9325번문제.py

```diff
@@ -0,0 +1,20 @@
+# 모든 옵션이 주어진 자동차를 구매하는데 필요한 액수를 계산해 주자.
+
+# 첫째 줄에 테스트 케이스의 개수가 주어진다.
+# 각 테스트 케이스의 첫 줄엔 자동차의 가격 s가 주어진다. (1 ≤ s ≤ 100 000)
+# 둘째 줄엔 해빈이가 구매하려고 하는 서로 다른 옵션의 개수 n이 주어진다. (0 ≤ n ≤ 1 000)
+# 뒤이어 n개의 줄이 입력으로 들어온다.
+# 각 줄은 q 와 p로 이루어져 있는데 q는 해빈이가 사려고 하는 특정 옵션의 개수이고 p는 해당 옵션의 가격이다. (1 ≤ q ≤ 100, 1 ≤ p ≤ 10 000)
+
+T = int(input())
+
+for _ in range(T):
+    s = int(input())
+    n = int(input())
+    price = s
```

Comment:
```suggestion
    price = int(input())
    n = int(input())
```
따로 s가 사용되지 않는다면 처음부터 price로 받아도 될 것 같습니다.